tests/js/setup.ts

```diff
@@ -3,9 +3,9 @@
 import {TextDecoder, TextEncoder} from 'util';
 
 import {InjectedRouter} from 'react-router';
-import {configure} from '@testing-library/react'; // eslint-disable-line no-restricted-imports
+import {configure as configureRtl} from '@testing-library/react'; // eslint-disable-line no-restricted-imports
 import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
-import Enzyme from 'enzyme'; // eslint-disable-line no-restricted-imports
```

Comment:
This is also an eslint error in eslint 6/7 since enzyme doesn't have a default export like this